docs/user-guide/migrate-to-greptimedb/migrate-from-mysql.md

```diff
@@ -0,0 +1,74 @@
+# Migrate from MySQL
+
+This document will guide you through the migration process from MySQL to GreptimeDB.
+
+## Before you start the migration
+
+Please be aware that though GreptimeDB supports the wire protocol of MySQL, it does not mean GreptimeDB implements all
+MySQL's features. You may refer to the "[ANSI Compatibility](../../reference/sql/compatibility.md)" to see the
+constraints regarding using SQL in GreptimeDB.
+
+## Migration steps
+
+### Step 1: Create the databases and tables in GreptimeDB.
+
+GreptimeDB has its own SQL syntax for creating tables. We cannot simply reuse the table creation SQLs that are produced
+by MySQL. So first please refer to our document for "[CREATE](../../reference/sql/create.md)ing" the corresponding
+databases and tables in GreptimeDB.
```

Comment:
Needs more descriptions about the data model when migrating from MySQL to GreptimeDB. For example, how to choose the primary key and time index etc.

Reply:
see [749398f](https://github.com/GreptimeTeam/docs/pull/1241/commits/749398fe5e16be1bf3b7b42b84ab9f0f799d14d8)